src/main/java/me/ars/pokerbot/config/GameConfig.java

```diff
@@ -0,0 +1,8 @@
+package me.ars.pokerbot.config;
+
+public class GameConfig {
+    public Integer startStash;
+    public Integer ante;
+    public Integer bigBlind;
+    public Integer smallBlind;
```

Comment:
Do we need this? Small blinds are always big blind/2.

Reply:
I'm not sure. Normally the small blind is half the big blind. But I'm not sure if that's always the case or just some convention.